File: _old_CmdMents/TimesAndSeasons/FastOnYomKippur.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace CmdMents.TimesAndSeasons
{
    class FastOnYomKippur : AfflictYourselfOnYomKippur
    {
        public FastOnYomKippur()
        {
            base.AlternateText = "Anyone who does not fast on that day must be cut off from his people.";
            base.Book = CommandmentBook.Leviticus;
            base.CanBeCarriedOutToday = true;
            base.Chapter = 23;
            base.CommandmentType = CommandmentType.Positive;
            base.FollowedByChristians = CommandmentObedience.None;
            base.FollowedByMessianics = CommandmentObedience.Obeyed;
            base.FollowedByObservantJews = CommandmentObedience.Obeyed;
            base.Number = 95;
            base.ShortSummary = "Fast on Yom Kippur.";
            base.Text = "Anyone who does not deny himself on that day must be cut off from his people.";
            base.Verse = 29;
        }
    }
}

File: EtzMitzvot/Commandments/TimesAndSeasons/fastOnYomKippur.ts
///<reference path='../commandmentBase.ts' />

class FastOnYomKippur extends CommandmentBase {
    constructor() {
        super();

        this.shortSummary = "Fast on Yom Kippur";
        this.text = "Anyone who does not deny himself on that day must be cut off from his people.";
        this.book = TorahBook.Leviticus;
        this.chapter = 23;
        this.verse = 29;

        this.canBeCarriedOutToday = true;
        this.requiresLivingInIsrael = false;
        this.requiresTemple = false;
        this.commentary = null;

        this.type = CommandmentType.Positive;
        this.christianObservance = CommandmentObedience.None;
        this.messianicObservance = CommandmentObedience.Binding;
        this.jewishObservance = CommandmentObedience.Binding;
        this.number = 95;

        this.children = [

        ];
    }
}  